convert/jenkinsjson/json/notification_test.go

```diff
@@ -0,0 +1,44 @@
+package json
+
+import (
+	"testing"
+
+	harness "github.com/drone/spec/dist/go"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestConvertNotification(t *testing.T) { // Convert bytes to string
+
+	var tests []runner
+	tests = append(tests, prepare(t, "/notification/notification_snippet", &harness.Step{
+		Id:   "notifyEndpointsc1ca8f",
+		Name: "Notification",
+		Type: "plugin",
+		Spec: &harness.StepPlugin{
+			Image: "plugins/webhook",
+			With: map[string]interface{}{
+				"urls":         []string{"https://webhook.site/9ffae84b-a338-43ef-9283-319d70574bf4"},
+				"method":       "POST",
+				"username":     "<+input>",
+				"password":     "<+input>",
+				"token-value":  "<+input>",
+				"token-type":   "<+input>",
```

Comment:
we don't need to mention both password and token at the same time - can you please update it to just token and remove username and password